src/FacultyDirectory/Controllers/UsersController.cs

```diff
@@ -1,42 +1,28 @@
 using System;
-using System.IdentityModel.Tokens.Jwt;
+using System.Linq;
 using System.Security.Claims;
+using FacultyDirectory.Core.Data;
+using FacultyDirectory.Core.Domain;
 using System.Text;
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Mvc;
-using Microsoft.IdentityModel.Tokens;
+using Microsoft.EntityFrameworkCore;
 
 namespace FacultyDirectory.Controllers
 {
     [ApiController]
     [Route("api/[controller]")]
     public class UsersController : Controller
```

Comment:
You'll need to add an authorize attribute on this controller now, or else anybody could add themselves as a user.  See one of the other controllers for what it looks like.